[2024-06-20T09:01:11.089+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T09:01:11.188+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.raw_orders.test manual__2024-06-20T08:59:25.665849+00:00 [queued]>
[2024-06-20T09:01:11.226+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.raw_orders.test manual__2024-06-20T08:59:25.665849+00:00 [queued]>
[2024-06-20T09:01:11.227+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-20T09:01:11.274+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtTestLocalOperator): transform.raw_orders.test> on 2024-06-20 08:59:25.665849+00:00
[2024-06-20T09:01:11.297+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4682) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T09:01:11.300+0000] {standard_task_runner.py:63} INFO - Started process 4694 to run task
[2024-06-20T09:01:11.298+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'ELT-PROJECT', 'transform.raw_orders.test', 'manual__2024-06-20T08:59:25.665849+00:00', '--job-id', '306', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpiaoxsctl']
[2024-06-20T09:01:11.305+0000] {standard_task_runner.py:91} INFO - Job 306: Subtask transform.raw_orders.test
[2024-06-20T09:01:11.379+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-06-20T09:01:11.581+0000] {task_command.py:426} INFO - Running <TaskInstance: ELT-PROJECT.transform.raw_orders.test manual__2024-06-20T08:59:25.665849+00:00 [running]> on host airflow-scheduler
[2024-06-20T09:01:12.147+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Data-Ninja' AIRFLOW_CTX_DAG_ID='ELT-PROJECT' AIRFLOW_CTX_TASK_ID='transform.raw_orders.test' AIRFLOW_CTX_EXECUTION_DATE='2024-06-20T08:59:25.665849+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-20T08:59:25.665849+00:00'
[2024-06-20T09:01:12.152+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T09:01:12.210+0000] {local.py:174} INFO - Could not import dbtRunner. Falling back to subprocess for invoking dbt.
[2024-06-20T09:01:12.217+0000] {local.py:292} INFO - Cloning project to writable temp directory /tmp/tmpftmnfno8 from /opt/airflow/dags/dbt/dbt_project
[2024-06-20T09:01:12.237+0000] {local.py:303} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/ELT-PROJECT__transform/target/partial_parse.msgpack
[2024-06-20T09:01:12.407+0000] {base.py:84} INFO - Using connection ID 'snow_conn' for task execution.
[2024-06-20T09:01:12.410+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-06-20T09:01:12.421+0000] {base.py:225} INFO - Using real values for profile dbt_project
[2024-06-20T09:01:12.432+0000] {config.py:291} INFO - Creating temporary profiles.yml with use_mock_values=False at /tmp/tmpaod27v2u/profiles.yml with the following contents:
dbt_project:
    outputs:
        dev:
            account: sk84886.europe-west2.gcp
            database: ELT_PROJECT
            password: '{{ env_var(''COSMOS_CONN_SNOWFLAKE_PASSWORD'') }}'
            role: SYSADMIN
            schema: PUBLIC
            type: snowflake
            user: DATANINJA
            warehouse: TRANSFORM
    target: dev

[2024-06-20T09:01:12.435+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmpftmnfno8', '--profiles-dir', '/tmp/tmpaod27v2u', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmpftmnfno8
[2024-06-20T09:01:12.440+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-06-20T09:01:12.443+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmpftmnfno8', '--profiles-dir', '/tmp/tmpaod27v2u', '--profile', 'dbt_project', '--target', 'dev']
[2024-06-20T09:01:12.486+0000] {subprocess.py:84} INFO - Command output:
[2024-06-20T09:01:24.476+0000] {subprocess.py:94} INFO - [0m09:01:24  Running with dbt=1.8.2
[2024-06-20T09:01:26.201+0000] {subprocess.py:94} INFO - [0m09:01:26  Updating lock file in file path: /tmp/tmpftmnfno8/package-lock.yml
[2024-06-20T09:01:26.273+0000] {subprocess.py:94} INFO - [0m09:01:26  Installing dbt-labs/dbt_utils
[2024-06-20T09:01:27.239+0000] {subprocess.py:94} INFO - [0m09:01:27  Installed from version 0.8.6
[2024-06-20T09:01:27.246+0000] {subprocess.py:94} INFO - [0m09:01:27  Updated version available: 1.2.0
[2024-06-20T09:01:27.259+0000] {subprocess.py:94} INFO - [0m09:01:27
[2024-06-20T09:01:27.266+0000] {subprocess.py:94} INFO - [0m09:01:27  Updates available for packages: ['dbt-labs/dbt_utils']
[2024-06-20T09:01:27.268+0000] {subprocess.py:94} INFO - Update your versions in packages.yml, then run dbt deps
[2024-06-20T09:01:33.881+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-06-20T09:01:33.883+0000] {local.py:254} INFO - Update your versions in packages.yml, then run dbt deps
[2024-06-20T09:01:33.885+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'test', '--models', 'raw_orders', '--project-dir', '/tmp/tmpftmnfno8', '--profiles-dir', '/tmp/tmpaod27v2u', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmpftmnfno8
[2024-06-20T09:01:33.888+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-06-20T09:01:33.890+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'test', '--models', 'raw_orders', '--project-dir', '/tmp/tmpftmnfno8', '--profiles-dir', '/tmp/tmpaod27v2u', '--profile', 'dbt_project', '--target', 'dev']
[2024-06-20T09:01:33.917+0000] {subprocess.py:84} INFO - Command output:
[2024-06-20T09:01:46.217+0000] {subprocess.py:94} INFO - [0m09:01:46  Running with dbt=1.8.2
[2024-06-20T09:01:48.815+0000] {subprocess.py:94} INFO - [0m09:01:48  Registered adapter: snowflake=1.8.3
[2024-06-20T09:01:52.375+0000] {subprocess.py:94} INFO - [0m09:01:52  Found 20 models, 31 data tests, 11 sources, 682 macros
[2024-06-20T09:01:52.386+0000] {subprocess.py:94} INFO - [0m09:01:52
[2024-06-20T09:01:59.379+0000] {subprocess.py:94} INFO - [0m09:01:59  Concurrency: 1 threads (target='dev')
[2024-06-20T09:01:59.381+0000] {subprocess.py:94} INFO - [0m09:01:59
[2024-06-20T09:01:59.401+0000] {subprocess.py:94} INFO - [0m09:01:59  1 of 1 START test not_null_raw_orders_ORDERID .................................. [RUN]
[2024-06-20T09:02:01.133+0000] {subprocess.py:94} INFO - [0m09:02:01  1 of 1 PASS not_null_raw_orders_ORDERID ........................................ [[32mPASS[0m in 1.73s]
[2024-06-20T09:02:01.140+0000] {subprocess.py:94} INFO - [0m09:02:01
[2024-06-20T09:02:01.141+0000] {subprocess.py:94} INFO - [0m09:02:01  Finished running 1 test in 0 hours 0 minutes and 8.75 seconds (8.75s).
[2024-06-20T09:02:01.363+0000] {subprocess.py:94} INFO - [0m09:02:01
[2024-06-20T09:02:01.365+0000] {subprocess.py:94} INFO - [0m09:02:01  [32mCompleted successfully[0m
[2024-06-20T09:02:01.366+0000] {subprocess.py:94} INFO - [0m09:02:01
[2024-06-20T09:02:01.368+0000] {subprocess.py:94} INFO - [0m09:02:01  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-06-20T09:02:07.724+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-06-20T09:02:07.725+0000] {local.py:254} INFO - [0m09:02:01  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-06-20T09:02:07.763+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v12.json is above dbt-ol supported version 7. This might cause errors.
[2024-06-20T09:02:07.764+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/run-results/v6.json is above dbt-ol supported version 5. This might cause errors.
[2024-06-20T09:02:07.777+0000] {local.py:349} INFO - Inlets: [Dataset(uri='snowflake://sk84886.europe-west2.gcp/ELT_PROJECT.raw.raw_orders', extra=None)]
[2024-06-20T09:02:07.778+0000] {local.py:350} INFO - Outlets: []
[2024-06-20T09:02:07.781+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-20T09:02:08.085+0000] {dag.py:3954} INFO - Setting next_dagrun for ELT-PROJECT to None, run_after=None
[2024-06-20T09:02:08.147+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T09:02:08.210+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ELT-PROJECT, task_id=transform.raw_orders.test, run_id=manual__2024-06-20T08:59:25.665849+00:00, execution_date=20240620T085925, start_date=20240620T090111, end_date=20240620T090208
[2024-06-20T09:02:08.289+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-20T09:02:08.388+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-20T09:02:08.392+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
